integration_tests/dockerfiles/hermes/build.sh

```diff
@@ -8,5 +8,13 @@ if [[ "$CI" == "true" ]]; then
 else
     VERSION=":$VERSION"
 fi
-docker build . -t ${ORG}hermes-test${VERSION}
+ARCH=$(uname -m)
+
+if [ "$ARCH" = "arm64" ] || [ "$ARCH" = "aarch64" ]; then
+  DOCKERFILE="Dockerfile.aarch64"
+else
+  DOCKERFILE="Dockerfile.x86_64"
+fi
```

Comment:
btw the only difference between these two containers are in base image (ubuntu:24.04 vs ubuntu:20.04)
Maybe we can build docker based on common platform and remove docker containers selection?